db.go

```diff
@@ -6,13 +6,17 @@ import (
 	"github.com/lightningnetwork/lnd/lnrpc"
 	"gorm.io/driver/postgres"
 	"gorm.io/gorm"
+	"gorm.io/gorm/logger"
 )
 
 func OpenDB(config *Config) (db *gorm.DB, err error) {
 	db, err = gorm.Open(postgres.Open(config.DatabaseUri), &gorm.Config{})
 	if err != nil {
 		return nil, err
 	}
+	//don't print warnings for slow sql
+	//because we use db transactions that span the rabbitmq publish operation
```

Comment:
why is publishing so slow?

Reply:
I think because of the publisher confirmation

Reply:
but how slow can that be? how slow is publishing a rabbitmq message?

Reply:
On testnet I had to increase the timeout from 1 to 3 seconds. We'll have to see in production how long it takes.

Reply:
hm, hard for me to believe that anything should take 1 to 3 seconds here. 
what is it doing? - that's just publishing the message, isn't it? 
it would mean that this is quite some bottleneck, no?

Reply:
to be investigated